src/queries/eds/rum-pageviews-cwv-count-threshold.sql

```diff
@@ -0,0 +1,106 @@
+--- description: Get pageviews for a given URL or domain having a Core Web Vital events count above a given threshold
+--- Access-Control-Allow-Origin: *
+--- interval: 30
+--- offset: 0
+--- startdate: 2022-02-01
+--- enddate: 2022-05-28
+--- timezone: UTC
+--- device: all
+--- url: -
+--- owner: -
+--- repo: -
+--- cwv_type: lcp
+--- cwv_count_threshold: 100
+--- sampling_noise_factor: 1000
+--- domainkey: secret
+
+
+WITH current_data AS (
+  SELECT * FROM
+    helix_rum.EVENTS_V3(
+      @url, -- domain or URL
+      CAST(@offset AS INT64), -- not used, offset in days from today
+      CAST(@interval AS INT64), -- interval in days to consider
+      @startdate, -- not used, start date
+      @enddate, -- not used, end date
+      @timezone, -- timezone
+      @device, -- device class
+      @domainkey
+    )
+),
+
+current_rum_by_id AS (
+  SELECT
+    id,
+    MAX(host) AS host,
+    MAX(user_agent) AS user_agent,
+    MAX(url) AS url,
+    MAX(CASE
+      WHEN @cwv_type = "lcp" THEN lcp
+      WHEN @cwv_type = "cls" THEN cls
+      WHEN @cwv_type = "fid" THEN fid
+      WHEN @cwv_type = "inp" THEN inp
+    END) AS core_web_vital,
+    MAX(referer) AS referer,
+    MAX(weight) AS weight
+  FROM current_data
+  WHERE
+    url LIKE CONCAT("https://", @url, "%")
+    OR url LIKE CONCAT(
+      "https://%", @repo, "--", @owner, ".hlx%/"
+    ) OR (@url = "-" AND @repo = "-" AND @owner = "-")
+  GROUP BY id
+),
```

Comment:
This does not work/is superfluous. `helix_rum.EVENTS_V3` already filters by URL. And `@owner` and `@repo` aren't working.